eng/targets/Npm.Common.targets

```diff
@@ -66,7 +66,8 @@
 
   <Target Name="Test" Condition="'$(IsTestProject)' == 'true'">
     <Message Importance="High" Text="Running npm tests for $(MSBuildProjectName)" />
-    <Exec Command="npm $(NpmTestArgs)" IgnoreStandardErrorWarningFormat="true" />
+    <Exec  Condition="'$(DailyTests)' != 'true'" Command="npm $(NpmTestArgs)" IgnoreStandardErrorWarningFormat="true" />
+    <CallTarget Condition="'$(DailyTests)' == 'true'" Targets="RunBrowserTests" />
```

Comment:
Can you remove these changes from this file? It is re-used across all .npmproj files in the repo, which will includes more than just SignalR's npm projects.

If you need a completely different implementation of the Test target, you can override this by putting your own version of `<Target Name="Test">` in `SignalR.Npm.FunctionalTests.npmproj`. Put this target after the `<Import>` to Directory.Build.targets, and it will override the one from Npm.Common.targets.